test/passing/ocp_indent_options.ml.ref

```diff
@@ -0,0 +1,32 @@
+Warning: ocp-indent option "base" is unsupported.
```

Comment:
Do we really want to print a warning here ? I think having 24 lines of warning for every single file is way too noisy

Reply:
I think it is important to warn the user when an option is not supported (Is silently ignoring an option really what we want ?), and the messages are not printed when `quiet` is set.

Reply:
Quiet is not the default though.
In the current situation, someone using --ocp-indent-config can either use the quiet mode and never see useful information or always have warnings related to ocp-indent (we're unlikely to support all ocp-indent options anytime soon). I think it's fine to warn if the option is not recognized. I think it's also fine to say in the documentation that some option have no effect.